protokube/pkg/protokube/aws_volume.go

```diff
@@ -126,11 +125,7 @@ func (a *AWSVolumes) discoverTags() error {
 
 	a.clusterTag = clusterID
 
-	if *instance.PrivateDnsNameOptions.HostnameType == ec2.HostnameTypeResourceName {
```

Comment:
Could you fix this condition to test for the availability of an IPv6 address instead (as the line below) instead of reverting?